Tests/TestCase.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace JakubZapletal\Component\BankStatement\Tests;
+
+
+class TestCase extends \PHPUnit_Framework_TestCase
+{
+}
+
+/*
+ * Convers E_RECOVERABLE_ERROR to exception.
+ * - used for tests relying on type hinting exceptions
+ */
+set_error_handler(function($errno, $errstr, $errfile, $errline) {
```

Comment:
Aha, thanks. It's only tested in 5.4 and 5.5, since yesterday also in 5.3. I am goning to have a look this evening and merge it.